src/components/Table/README.md

```diff
@@ -251,12 +252,15 @@ const MyTable1 = withTableSettings({sortable: false})(Table);
 
 ### Properties
 
-| Name               | Description                     |                     Type                     |
-| :----------------- | :------------------------------ | :------------------------------------------: |
-| settingsPopupWidth | TableColumnSetup pop-up width   |                `number` `fit`                |
-| settings           | Current settings                |             `TableSettingsData`              |
-| updateSettings     | Settings update handle          | `(data: TableSettingsData) => Promise<void>` |
-| renderControls     | Allows to render custom actions |               `RenderControls`               |
+| Name                   | Description                                                  |                           Type                           |
+| :--------------------- | :----------------------------------------------------------- | :------------------------------------------------------: |
+| settingsPopupWidth     | TableColumnSetup pop-up width                                |                      `number` `fit`                      |
+| settings               | Current settings                                             |                   `TableSettingsData`                    |
+| updateSettings         | Settings update handle                                       |       `(data: TableSettingsData) => Promise<void>`       |
+| renderControls         | Allows to render custom actions                              |                     `RenderControls`                     |
+| filterPlaceholder      | Text that appears in the control when no search value is set |                         `string`                         |
+| filterEmptyPlaceholder | Text that appears when no one item is found                  |                         `string`                         |
+| filterItems            | Function for filtering items                                 | `(item: TableColumnSetupItem, value: string) => boolean` |
```

Comment:
All the props should have `settings` in their names to show belonging to the HOC. For example:
`filterSettings`
`settingsFilterPlaceholder`
`settingsFilterEmptyMessage` (to be consistent with `emptyMessage` of the `Table` props)

Reply:
[fixed](https://github.com/gravity-ui/uikit/pull/1627/commits/49395e65e53f303aa73968265b49bf54cd6c4d35)